javascript-cont/vscode-and-style.md

```diff
@@ -71,139 +71,260 @@ Hello Naya Spence!
 
-The `.eslintrc.json` file will contain information about the linting rules. Essentially they tell the Eslint plugin in VS Code what code style rules to enforce.
+The `package.json` file contains information about the project including the dependencies (libraries required) and scripts to run the app.
 
+The `package-lock.json` file primarily records the explicit version of each dependency installed. For example, if the `package.json` requests package `really-useful-package` of at least version 3, then the `package-lock.json` would record whether it was version 3.1, 3.2, or some other sub-version that was actually installed. Especially for projects involving more than one developer, this ensures that the same versions of dependencies are installed on all machines, minimizing isolated issues due to different versions of dependencies.
 
-### An Example Node Project
+### An Example Node.js Project
 
-Clone the [node-example](https://github.com/AdaGold/node-example) repository. Then run `yarn install` to install the dependencies.
+Clone the [node-example](https://github.com/AdaGold/node-example) repository. Then run `npm install` or `npm i` to install the dependencies. There's no need to fork first, unless you would like to try making changes and then be able to commit your work to your own repo.
 
-Examine the `package.json` file and look at the key-value pairs in the json file. 
+Examine the `package.json` file and look at the key-value pairs in the json file.
 
-![Example package.json](../assets/learning-another-language__vscode-and-style__package-json.png)
+![Example package.json highlighting several areas of the file. "name" and "version" are described as "Name and version number". "scripts" is described as "Scripts to run the application. In this case, we can run the app with: npm start". "devDependencies" is described as "Dependencies for the development process, not for production." "dependencies" is described as "Dependencies used in production".](../assets/learning-another-language__vscode-and-style__package-json.png)  
+*Fig. Several important areas of the `package.json` file.* [*(Full size image)*](../assets/learning-another-language__vscode-and-style__package-json.png)
 
-The `scripts` key contains a list of commands that can be run from the command line. For example, `yarn run start` will run the `start` script.
+The `scripts` key contains a list of commands that can be run from the command line. For example, `npm run start` runs the `start` script. A few script names are considered "well-known" and can be run without the `run` keyword. For example, `npm start` runs the `start` script. Other well-known scripts are `stop`, `restart`, and `test`. If we add a custom script entry not matching one of these names, then we must include the `run` keyword when running it.
```

Comment:
Since it's not essential to know that we can omit the `run` keyword for some common scripts, what do you think about putting that part into a callout-info block? Specifically this part:

"A few script names are considered "well-known" and can be run without the `run` keyword. For example, `npm start` runs the `start` script. Other well-known scripts are `stop`, `restart`, and `test`. If we add a custom script entry not matching one of these names, then we must include the `run` keyword when running it."